### ISSUE TITLE
Bazarr downloading same subtitle every few hours...

### ISSUE THREAD

**Issue by mobamoba** [morpheus65535/bazarr#120]
Bazarr is downloading the same subtitle over and over and never clearing it from the Wanted section nor fnding it on disk even though it's there. I've attached a screenshot.
![image](https://user-images.githubusercontent.com/11135869/43990984-7cf91bba-9d31-11e8-8a7c-ef2ce475b0f3.png)

I've tried both the Master and Development branches but there's some kind of bug that preventing Bazarr from noticing that it's already downloaded the subtitle and from noticing the subtitle exists during disk scans. Thanks.

**Comment by morpheus65535** [comment]
Can you please provide log file and screenshot of the main series pages? Thanks.

**Comment by mobamoba** [comment]
Attached.
[bazarr.log](https://github.com/morpheus65535/bazarr/files/2280246/bazarr.log)
![untitled](https://user-images.githubusercontent.com/11135869/43991873-8282dcb2-9d43-11e8-8cac-47b4112f5a13.png)

**Comment by morpheus65535** [comment]
Can you send me the subtitles file over Discord or another file sharing platform?

**Comment by mobamoba** [comment]
I figured out what Bazarr is doing that's causing the problem. The episode title is:
The Sinner - 2x02 - Part II

When Bazarr is renaming, it's calling the SRT:
The Sinner - 2x02 - Part Ii

Notice - instead of "II" (the actual name) Bazarr made the second "I" lowercase. This is why it's constantly downloading then failing to detect - it's renaming it improperly by not respecting the case of the second letter.

**Comment by mobamoba** [comment]
Not there. I searched the entire Python27 tree and couldn't find subliminal.exe anywhere.

**Comment by morpheus65535** [comment]
%appdata%\Roaming\Python\Scripts ?

**Comment by mobamoba** [comment]
Nope, nothing. I did a full computer search and there's no subliminal.exe anywhere. I found an srt.exe in one of the Python subdirectories but when I tried running the command you sent using that, it didn't work.

Is there a way you could test it at your end since I'm not having much luck finding it at mine?

**Comment by morpheus65535** [comment]
That's the issue, on my side I don't get this problem.

You can install subliminal by running "pip install subliminal". You should get the executable after that.

**Comment by mobamoba** [comment]
That worked. Subliminal found the properly named subtitle so the small letter must've appeared when Bazarr renamed it and moved it to the video directory.

**Comment by morpheus65535** [comment]
That's strange because Bazarr do not intervene in this process. Did you enable scenename in settings?

**Comment by mobamoba** [comment]
No. Do you want me to? When subliminal downloaded directly, it came out as he Sinner - 2x02 - Part II.eng.srt, so, at the very least, something got rid of "eng" in addition to making the "II" into "Ii"; what might be doing that?

**Comment by morpheus65535** [comment]
I really think it's an edge case that won't repeat. Never seen that. I would suggest to rename the .srt file to "II" and scan for files in Bazarr. Not the kind of resolution I normally use but in this case I don't think it worth the time to reproduce it (I still can't) and fix it. Are you ok with this?

**Comment by mobamoba** [comment]
I'm not sure it's an edge case only because this coming week's episode of The Sinner is called "III" so it may happen again. Is there a way to do a better tracelog to see exactly where in the process the file is being renamed incorrectly?

**Comment by morpheus65535** [comment]
Did you try to enable debug log level?

**Comment by mobamoba** [comment]
I didn't. How about if we leave this thread open and, if it happens again, I'll run a debug log and post it.

**Comment by morpheus65535** [comment]
No problem, just keep me informed next week!

**Comment by mobamoba** [comment]
It downloaded Sinner III so I guess we can close this thread - must've been an edge case as you said.
